packages/ods-react/src/components/pagination/src/components/pagination/Pagination.tsx

```diff
@@ -1,18 +1,20 @@
 import { Pagination as VendorPagination } from '@ark-ui/react/pagination';
 import classNames from 'classnames';
-import { type ComponentPropsWithRef, type FC, type JSX, forwardRef } from 'react';
+import { type ComponentPropsWithRef, type FC, type JSX, forwardRef, useState } from 'react';
 import { BUTTON_VARIANT, Button } from '../../../../button/src/index';
 import { ICON_NAME, Icon } from '../../../../icon/src/index';
 import { PaginationButtonWithTooltip } from '../pagination-button-with-tooltip/PaginationButtonWithTooltip';
 import { PaginationItem } from '../pagination-item/PaginationItem';
+import {
+  PaginationPageSizeSelector,
+  type RenderTotalItemsLabel,
+} from '../pagination-page-size-selector/PaginationPageSizeSelector.tsx';
```

Comment:
remove `tsx` (should be fixed by the field PR, so that it won't be set by default anymore)